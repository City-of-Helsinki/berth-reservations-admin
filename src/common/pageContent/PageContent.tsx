import React from 'react';
import classNames from 'classnames';
import styles from './pageContent.module.scss';

export interface PageContentProps {
  children: React.ReactNode;
  className?: string;
}

const PageContent = ({ children, className }: PageContentProps) => {
  return <div className={classNames(styles.pageContent, className)}>{children}</div>;
};

export default PageContent;
