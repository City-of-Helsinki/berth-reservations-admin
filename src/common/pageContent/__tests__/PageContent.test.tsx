import React from 'react';
import { shallow } from 'enzyme';

import PageContent from '../PageContent';

describe('PageContent', () => {
  it('renders normally', () => {
    const wrapper = shallow(
      <PageContent>
        <p>Test</p>
      </PageContent>
    );

    expect(wrapper.render()).toMatchSnapshot();
  });
});
