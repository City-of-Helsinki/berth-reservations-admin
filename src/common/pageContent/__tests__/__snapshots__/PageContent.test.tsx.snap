// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent renders normally 1`] = `
<div
  class="pageContent"
>
  <p>
    Test
  </p>
</div>
`;
